src/savegame_json.cpp

```diff
@@ -190,6 +190,17 @@ static void deserialize( weak_ptr_fast<monster> &obj, JsonIn &jsin )
     //    }
 }
 
+static tripoint read_legacy_creature_pos( const JsonObject &data )
+{
+    tripoint pos;
+    if( !data.read( "posx", pos.x ) ) {
+        debugmsg( "Bad Creature JSON: neither `location` nor `posx` found" );
+    }
+    data.read( "posy", pos.y );
+    data.read( "posz", pos.z );
```

Comment:
```suggestion
    if( !data.read( "posx", pos.x ) || !data.read( "posy", pos.y ) || !data.read( "posz", pos.z ) ) {
        debugmsg( R"(Bad Creature JSON: neither "location" nor "posx", "posy", "posz" found)" );
    }
```
Backticks don't really make sense in this context, but I understand why you chose them :).

Reply:
Thanks for the tip about R"()".
Done.